examples/tutorials/advanced/dataset_on_surface.py

```diff
@@ -0,0 +1,167 @@
+"""
+Draping a dataset on top of a topographic surface
+==================================================
+
+It can be visually appealing to "drape" a dataset over a topographic surface.
+This can be accomplished using the ``drapegrid`` parameter of
+:meth:`pygmt.Figure.grdview`.
+
+This tutorial consists of two examples:
```

Comment:
Please re-wrap the lines to 88 characters when possible.

Reply:
Adjusted the line length in commit https://github.com/GenericMappingTools/pygmt/pull/3316/commits/1038e8680d2adb4037a45404246e9f98de28e4e8.